octobot_trading/enums.py

```diff
@@ -328,22 +334,27 @@ class ExchangeConstantsOrderColumns(enum.Enum):
     SYMBOL = "symbol"
     QUANTITY_CURRENCY = "quantity_currency"
     TYPE = "type"
+    OCTOBOT_ORDER_TYPE = "octobot_order_type"
     SIDE = "side"
     PRICE = "price"
     AMOUNT = "amount"
     COST = "cost"
     AVERAGE = "average"
     FILLED = "filled"
+    FILLED_PRICE = "filled_price"
     REMAINING = "remaining"
     STATUS = "status"
     FEE = "fee"
+    FEES = "fees"
     TRADES = "trades"
     MAKER = "maker"
     TAKER = "taker"
     ORDER = "order"
-    TAKERORMAKER = "takerOrMaker"
-    REDUCE_ONLY = "reduceOnly"
-    STOP_PRICE = "stopPrice"
+    TAKER_OR_MAKER = "taker_or_maker"
```

Comment:
Can you redo your rebase :/ sorry we added more commits after your rebase

Reply:
its fixed